models/dao/transactions/base/dao_transactions_base_eth.sql

```diff
@@ -0,0 +1,172 @@
+ 
+        UNION ALL 
+
+        SELECT 
+            evt_block_time as block_time, 
+            evt_tx_hash as tx_hash, 
+            0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000 as token,
+            value, 
+            "to" as dao_wallet_address,
+            'tx_in' as tx_type,
+            evt_index as tx_index,
+            "from" as address_interacted_with,
+            array[evt_index] as trace_address
+        FROM 
+        {{ source('erc20_base', 'evt_transfer') }}
+        {% if not is_incremental() %}
+        WHERE evt_block_time >= DATE '{{transactions_start_date}}'
+        {% endif %}
+        {% if is_incremental() %}
+        WHERE evt_block_time >= date_trunc('day', now() - interval '7' Day)
+        {% endif %}
+        AND "to" IN (SELECT dao_wallet_address FROM dao_tmp)
+        AND contract_address = 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000
+
+        UNION ALL 
+
+        SELECT 
+            evt_block_time as block_time, 
+            evt_tx_hash as tx_hash, 
+            0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000 as token,
+            value, 
+            "from" as dao_wallet_address,
+            'tx_out' as tx_type,
+            evt_index as tx_index,
+            "to" as address_interacted_with,
+            array[evt_index] as trace_address
+        FROM 
+        {{ source('erc20_base', 'evt_transfer') }}
+        {% if not is_incremental() %}
+        WHERE evt_block_time >= DATE '{{transactions_start_date}}'
+        {% endif %}
+        {% if is_incremental() %}
+        WHERE evt_block_time >= date_trunc('day', now() - interval '7' Day)
+        {% endif %}
+        AND "from" IN (SELECT dao_wallet_address FROM dao_tmp)
+        AND contract_address = 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000
+)
+
+SELECT 
+    dt.blockchain,
+    dt.dao_creator_tool, 
+    dt.dao, 
+    dt.dao_wallet_address, 
+    CAST(date_trunc('day', t.block_time) as DATE) as block_date, 
+    CAST(date_trunc('month', t.block_time) as DATE) as block_month, 
+    t.block_time, 
+    t.tx_type,
+    t.token as asset_contract_address,
+    'ETH' asset,
+    CAST(t.value AS double) as raw_value, 
+    t.value/POW(10, 18) as value, 
+    t.value/POW(10, 18) * p.price as usd_value,
+    -- COALESCE(p.price, dp.median_price) as usd_value, 
+    t.tx_hash, 
+    t.tx_index,
+    t.address_interacted_with,
+    t.trace_address
+FROM 
+transactions t 
+INNER JOIN 
+dao_tmp dt 
+    ON t.dao_wallet_address = dt.dao_wallet_address
+LEFT JOIN 
+{{ source('prices', 'usd') }} p 
+    ON p.minute = date_trunc('minute', t.block_time)
+    AND p.symbol = 'WETH'
+    AND p.blockchain = 'base'
+    {% if not is_incremental() %}
+    AND p.minute >= DATE '{{transactions_start_date}}'
+    {% endif %}
+    {% if is_incremental() %}
+    AND p.minute >= date_trunc('day', now() - interval '7' Day)
+    {% endif %}
+-- LEFT JOIN 
+-- {{ ref('dex_prices') }} dp 
```

Comment:
similar idea here

Reply:
corrected ser.. thank you.. for some weird reasons I can't resolve the conflicts... nothing serious because I edited on the same line...

I'm resolving from GitHub here and after I finish resolving.. when I click merge conflicts... it says out of date.